lib/unexpectedMitm.js

```diff
@@ -730,6 +730,46 @@ module.exports = {
                     return recordedExchanges;
                 });
             })
+            .addAssertion('<any> with http mocked out [and verified] [returning the file] <string> <assertion>', function (expect, subject, filePathWithoutExt) {
+                expect.errorMode = 'nested';
+                var testFile = filePathWithoutExt + '.js';
+
+                if (process.env.UNEXPECTED_MITM_WRITE === 'true') {
+                    return executeMitm(expect, subject).then(function (recordedExchanges) {
+                        var output = 'module.exports = ' +  stringify(recordedExchanges) + ';\n';
```

Comment:
Please supply the second argument to `stringify` (indentationWidth). I just made a recording that ended up with a 1 space indent :)